Assignment4/JulioMartinezB-e22a329/Task6.py

```diff
@@ -0,0 +1,73 @@
+# -*- coding: utf-8 -*-
+"""Task06.ipynb
+Automatically generated by Colaboratory.
+Original file is located at
+    https://colab.research.google.com/drive/1hSv61ejXESrp34CUz1PqZCsoA43tzc8y
+**Task 06: Modifying RDF(s)**
+"""
+
+"!pip install rdflib "
```

Comment:
you cannot leave strings hanging around in python. It won't work

Reply:
that did not give me any problems, but I have already removed it.